mig-frankfurt/dataelementhub.dal
# #2 - Change unique constraint on user namespace grants table

**What's in the PR**
* renamed user_namespace_grants to user_namespace_access

**How to test manually**
* Ideally...take an existing dehub database and clone it (CREATE DATABASE _newdb_ WITH TEMPLATE _olddb_ OWNER _olddbuser_)
* use the newly created db in pom.xml db settings
* call mvn clean install flyway:migrate jooq-codegen:generate clean install
* check if the table is "renamed" and all your entries match the old ones
